src/configuration.ts

```diff
@@ -393,6 +393,7 @@ export function setConfigurationCachePath(path: string): void { configurationCac
 // and when the settings watcher detects a change of any properties that may impact the dryrun output.
 export async function readConfigurationCachePath(): Promise<void> {
     // how to get default from package.json to avoid problem with 'undefined' type?
```

Comment:
Is this comment stale?

Reply:
I don't remember what I meant by that. Thanks for pointing it out. I removed the 6 occurrences of it.